### DISCUSSION TITLE
Send custom data to a Plugin

### DISCUSSION THREAD

**Discussion by hardeath950** [ONLYOFFICE/DocumentServer#1815] [Q&A] [answered]
is it possible to send custom data to a plugin, when instantiating a DocsAPI?

I'm trying to create a Search And Replace Plugin, which receives its "Dictionary" by parameters when starting

**Comment by ShockwaveNN** [comment, marked as answer]
Hi, I'm not very good at plugins, but seems [that ](https://github.com/ONLYOFFICE/sdkjs-plugins/blob/master/example_add_content_controls/scripts/code.js#L97) plugin recieve some data in `init` method by some param

Not sure that it's that are you looking for, but maybe some other plugin examples in this repo has something that can help you
